examples/Tweet.hs

```diff
@@ -108,7 +97,10 @@ main = runBH' $ do
   let boost = Nothing
```

Comment:
Ah nice mapping fix makes sense, I only wonder why the mapping worked with the `   _ <- putMapping testIndex testMapping TweetMapping
`